src/MercadoPago/Net/MPResponse.php

```diff
@@ -12,7 +12,7 @@ class MPResponse
      */
     public function __construct(
         private int $status_code,
-        private array $content
+        private ?array $content
```

Comment:
Complementando a discussão, se optarmos por retornar um `bool` no delete como sugeri acima, também resolvemos esse problema, pois não haverá necessidade de buscar o `content` do `MPResponse`.
Desse modo, podemos deixar como estava antes, com o content sendo obrigatório.
```suggestion
        private array $content
```